CHANGES.md

```diff
@@ -2,6 +2,11 @@
 
 ## __NEXT__
 
+### Features
+
+* Add the curate subcommand with two sub-subcommands, passthru and normalize-strings [#1039][] (@joverlee521)
```

Comment:
A little jarring how a +1,921 -4 diff can be reduced to a single sentence! :-)

It wouldn't be unwarranted to say a bit more about `augur curate` here (goals, planned functionality, reason for existing, etc.) if you wanted.

Reply:
Suggestion: add a file `docs/usage/cli/curate.rst` and link to the generated doc page from here.

Reply:
Thanks both! I add docs and a bit more to the changelog.

Reply:
I added the usage docs for augur curate with subdirectory that contains a separate page for each sub-subcommand since we are expecting to add more commands (see [preview](https://nextstrain--1039.org.readthedocs.build/projects/augur/en/1039/usage/cli/curate/)). 

I also added more to the changelog to link to the new usage docs and also included more features that are all packaged into this PR.

Reply:
(I didn't add API docs to avoid conflicts with https://github.com/nextstrain/augur/pull/1087)